### ISSUE TITLE
[Bug?] Handling generic types

### ISSUE THREAD

**Issue by KidyovrS** [tgreyjs/typedoc-plugin-markdown#21]
Hello,

There seems a bug when this plugin handles generic types such as `MyClass<T>`.

I have created a repository for your reference [here](https://github.com/KidyovrS/typedoc-markdown-sample).

Code:
```ts
export class MyFirstClass {
}

export class MySecondClass<T> {
}

export class MyThirdClass {
    constructor(public arg: Array<MySecondClass<MyFirstClass>>) {
        // do nothing here
    }
}
```

`md` file generated (or you can see it [here](https://github.com/KidyovrS/typedoc-markdown-sample/blob/master/out_markdown/classes/_index_.mythirdclass.md)):
![image](https://user-images.githubusercontent.com/38679621/39166922-de55ee4c-47be-11e8-946d-9a8adf19cee9.png)

You can see that type of `arg` is rendered incorrectly.

Here's the html result by `typedoc` without plugin:
![image](https://user-images.githubusercontent.com/38679621/39167000-3ef9d542-47bf-11e8-8459-31d83d1ec59a.png)

It seems that this should be the correct result.

Thanks,
KidyovrS

**Comment by KidyovrS** [comment]
@tgreyjs Thanks for the quick fix.

**Comment by tgreyjs** [comment]
No worries - sorry only just published - its version 1.1.5
